app/releases/page.tsx

```diff
@@ -50,19 +50,25 @@ export default async function Page() {
 
             <div className="flex flex-col p-6">
               <h3 className={`font-semibold`}>{release.name}</h3>
-              <p className="text-sm text-secondary-700 dark:text-secondary-300">
+              <p className="text-xs text-secondary-700 dark:text-secondary-300 sm:text-sm">
                 Released by{" "}
                 <a
                   href={`https://github.com/${release.author.login}`}
                   target="_blank"
-                  className="font-semibold"
+                  className="text-sm font-semibold sm:text-base"
                 >
                   {release.author.login}
                 </a>{" "}
+                on{" "}
+                {new Date(release.createdAt).toLocaleDateString("en-US", {
+                  day: "2-digit",
+                  month: "short",
+                  year: "numeric",
+                })}
```

Comment:
Use the custom `formatDate` utility function instead, so that we have consistent date formats across the site.

Reply:
okk I will make changes and update you , and I have used that format as it is also used in the Release section in homepage

Reply:
nice find, update there too 😄

Reply:
okk , ill update it

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/146672184/52904469-608c-4abf-9367-3cc012065940)
![image](https://github.com/coronasafe/leaderboard/assets/146672184/82700e6b-e756-4055-8954-6edf53e5e218)

how does this look ? i have displayed only the git hub icon when dimension is less than 370px , and also I have one doubt like is it not necessary to put the version number as we do previously before shifting to octokit ?

Reply:
Yes, it is important to show the version number.

Reply:
Currently i guess it is not fetched , can i take up this issue and solve ?

Reply:
yup

Reply:
Should i solve that part too under this issue ?

Reply:
Create a new issue, but could be solved in the same PR by linking multiple issues to it

Reply:
okk